src/region/RegionAnnotations.tsx

```diff
@@ -60,10 +60,11 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         setStatus(CreatorStatus.started);
     };
 
-    handleStop = (shape: Rect): void => {
+    handleStop = (shape: Rect | null): void => {
         const { location, setStaged, setStatus } = this.props;
-        setStaged({ location, shape });
-        setStatus(CreatorStatus.staged);
+
+        setStaged(shape ? { location, shape } : null);
+        setStatus(shape ? CreatorStatus.staged : CreatorStatus.cancel);
```

Comment:
Can we set it `CreatorStatus.init` so that we don't need to introduce a new status `CreatorStatus.cancel`?

Reply:
If we set it to CreatorStatus.init, then we have to introduce a new AnnotationInput type for AnnotationInput.INIT. Is that okay?

Reply:
No, you don't. You've already emitted `cancel` in `eventing/staged.ts`